### PR TITLE
Fix last entity not loaded

### PR DESCRIPTION
**Pull request by Amoki**
Fix side bug of https://github.com/xeokit/xeokit-sdk/issues/152

### CODE REVIEW

File: src/plugins/XKTLoaderPlugin/XKTLoaderPlugin.js [hunk 1]
```diff
@@ -855,7 +855,7 @@ class XKTLoaderPlugin extends Plugin {
                 const lastEntity = (i === numEntities - 1);
                 const meshIds = [];
 
-                for (let j = entityMeshes [i], jlen = lastEntity ? entityMeshes.length : entityMeshes [i + 1]; j < jlen; j++) {
+                for (let j = entityMeshes [i], jlen = lastEntity ? entityMeshIds.length : entityMeshes [i + 1]; j < jlen; j++) {
```
Comment (tmarti) [inline src/plugins/XKTLoaderPlugin/XKTLoaderPlugin.js]: This leed needs NOT to be changed, as `entityMeshIds` only exists for XKT v2, and this line corresponds to XKT v1

**Comment by tmarti** [comment]
See the comment, but the change in line 858 must NOT be done.

The bug is for XKT v2 loading, and that line is part of XKT v1 loading.

The proper change is the one at line 735

**Comment by Amoki** [comment]
Fixed!

**Comment by barnabasmolnar** [comment]
Guys, there's a closing curly brace missing before the final export in the xkt loader plugin file, on line 896 I believe. 

As is, the XKTLoaderPlugin is broken atm. :(

**Comment by Amoki** [comment]
My bad. A fix is here: https://github.com/xeokit/xeokit-sdk/pull/165

**Pull request merged**